pkgs/servers/plex/default.nix

```diff
@@ -1,81 +1,51 @@
-{ stdenv, fetchurl, rpmextract, glibc
-, dataDir ? "/var/lib/plex" # Plex's data directory must be baked into the package due to symlinks.
-, enablePlexPass ? false
-}:
+{ stdenv, fetchurl, buildFHSUserEnv, runCommand, writeScript, rpmextract }:
 
 let
-  plexPass = throw "Plex pass has been removed at upstream's request; please unset nixpkgs.config.plex.pass";
-  plexpkg = if enablePlexPass then plexPass else {
-    version = "1.13.0.5023";
-    vsnHash = "31d3c0c65";
-    sha256 = "b94d571bcd47bc3201fd7f54b910f5693cabd1b543d5d0b4705668b7b7c24602";
-  };
-
-in stdenv.mkDerivation rec {
-  name = "plex-${version}";
-  version = plexpkg.version;
-  vsnHash = plexpkg.vsnHash;
-  sha256 = plexpkg.sha256;
+  name = "plexmediaserver-${version}";
+  version = "1.9.6.4429-23901a099";
```

Comment:
Only thing I noticed, this is a major step back in terms of version.

The latest publicly available release works fine.

```diff
diff --git a/pkgs/servers/plex/default.nix b/pkgs/servers/plex/default.nix
index 4ce1410e46a..247eb7ad737 100644
--- a/pkgs/servers/plex/default.nix
+++ b/pkgs/servers/plex/default.nix
@@ -2,11 +2,11 @@

let
  name = "plexmediaserver-${version}";
-  version = "1.9.6.4429-23901a099";
+  version = "1.13.2.5154-fd05be322";                                                                                                                                                       

  src = fetchurl {                                                                                                                                                                              url = "https://downloads.plex.tv/plex-media-server/${version}/${name}.x86_64.rpm";
-    sha256 = "0bmqf8b2d9h2h5q3n4ahs8y6a9aihj63rch7wd82rcr1l9xnqk9d";
+    sha256 = "09hy9xhhv17jbzplyls13xrzaxndlc278amp6k3w8q4j6wpsi6np";
  };                                                                                                                    
```